std/net/curl.d

```diff
@@ -2304,18 +2304,26 @@ private bool decodeLineInto(Terminator, Char = char)(ref const(ubyte)[] basesrc,
 /**
   * HTTP client functionality.
   *
-  * Example:
   * ---
   * import std.net.curl, std.stdio;
   *
   * // Get with custom data receivers
```

Comment:
For consistency this comment should be moved out of the code block too. Then the first example will look like the two examples below.